src/com/google/javascript/jscomp/ProcessEs6Modules.java

```diff
@@ -110,13 +106,21 @@ public ProcessEs6Modules(Compiler compiler, ES6ModuleLoader loader,
     this.reportDependencies = reportDependencies;
   }
 
-  public void processFile(Node root) {
+  /**
+   * If a file contains an ES6 "import" or "export" statement, or the forceRewrite
+   * option is true, rewrite the source as a module.
+   *
+   * @param root
+   * @param forceRewrite
```

Comment:
Leave these off unless there is more to say about them. No requirement in Java to have a `@param` for every param.